### ISSUE TITLE
Can the influx shell tell you if you are _successfully_ logged in or not?

### ISSUE THREAD

**Issue by viktorsmari** [influxdata/docs.influxdata.com#1266]
When you do 
`influx -username 'someuser' -password 'somepass' -database 'somedb'`
you get the response:

    Connected to http://localhost:8086 version 1.3.5
    InfluxDB shell 0.10.0

How can I verify if the password is correct or not?

If I then do 
`show databases`  I get:

    ERR: error authorizing query: no user provided

Now I always have to do `show databases` just to see if the command works.

I would be **nice to have** some indication if you are logged in, or not.

Something like:

    Logged in as 'user'

**Comment by timhallinflux** [comment]
Hi @viktorsmari -- I'd encourage you to update the InfluxDB Shell to a more current edition. (0.10.0) is getting quite old now.

This particular GitHub repo is for documentation related issues.   But, it feels like this is more of a request for an enhancement for the InfluxDB shell itself.  But, I would encourage you to file this ticket over here: https://github.com/influxdata/influxdb/issues

You might also consider setting the username and password as environment variables.  Once that is done correctly once...you won't have to worry about re-entering them over and over.

The root of the problem is that the shell doesn't attempt to validate the password until the first command is run.   

I'm going to close this issue here...but, I agree with you that this would be a valuable addition to the shell.
